### DISCUSSION TITLE
Windows image build fails during execution of "Install-Wix.ps1" script

### DISCUSSION THREAD

**Discussion by pbk8s** [actions/virtual-environments#11339] [General questions]
### Description

While running the Windows image build for Windows 11 Arm64 using image SKU (win11-24h2-ent), the script "Install-Wix.ps1" fails with the following error: 
`The install of DotNet3.5 was NOT successful.
    azure-arm.image: Error while running 'C:\ProgramData\chocolatey\lib\DotNet3.5\Tools\ChocolateyInstall.ps1'.
    azure-arm.image:  See log for details.
    azure-arm.image: Failed to install wixtoolset because a previous dependency failed.
    azure-arm.image:
    azure-arm.image: Chocolatey installed 0/2 packages. 2 packages failed.
    azure-arm.image:  See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
    azure-arm.image:
    azure-arm.image: Failures
    azure-arm.image:  - DotNet3.5 (exited 1) - Error while running 'C:\ProgramData\chocolatey\lib\DotNet3.5\Tools\ChocolateyInstall.ps1'.
    azure-arm.image:  See log for details.
    azure-arm.image:  - wixtoolset - Failed to install wixtoolset because a previous dependency failed.
    azure-arm.image: Running [#2]: choco install wixtoolset -y --force
`
It has been successful before, no changes were made and it still fails. Tried it on an older version of Windows 11-Arm64 but still the same result.

### Platforms affected

- [ ] Azure DevOps
- [ ] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] Ubuntu 24.04
- [ ] macOS 12
- [ ] macOS 13
- [ ] macOS 13 Arm64
- [ ] macOS 14
- [ ] macOS 14 Arm64
- [ ] macOS 15
- [ ] macOS 15 Arm64
- [ ] Windows Server 2019
- [X] Windows Server 2022

### Image version and build link

  image_offer                            = "windows11preview-arm64"
  image_publisher                        = "microsoftwindowsdesktop"
  image_sku                              = "win11-24h2-ent"

### Is it regression?

yes

### Expected behavior

The Build should succeed. It has been built before.

### Actual behavior

The build fails after multiple retries.

### Repro steps

Run the latest Windows image build on Windows 11-Arm64 VM in Azure.

**Comment by vidyasagarnimmagaddi** [comment]
Hi @pbk8s , Thank you for bringing this issue to our attention. We are looking into this issue and will update you on this issue after investigating.

**Comment by hemanthmanga** [comment]
Hi @pbk8s , Could you please confirm if you are using self-hosted runners?

**Comment by pbk8s** [comment]
@hemanthmanga I am building a Windows based image using Windows 11 Arm64 ISO in Azure. This is to create a runner image based on Windows 11-Arm64.

**Comment by hemanthmanga** [comment]
Hi @pbk8s , 
As per the issue, we are not facing any problems with our Windows images. We are moving this issue to the discussion for further progress.

If possible, could you please share your repository code?

**Comment by pbk8s** [comment]
@hemanthmanga Here's my fork of the main runner-images repository. 
[Runner Images Repo](https://github.com/pbk8s/runner-images)
I am still facing the same issue, ran the build again. It fails at the same step. 
I am running this build on a "D2Ps_v5" VM in Azure.

**Reply by pbk8s** [reply]
@hemanthmanga did you get a chance to look at the repo. I am still blocked because of this.

**Comment by kishorekumar-anchala** [comment]
Hi @pbk8s - if it is belongs to ARM64 , kindly raise a concern with [partner-images](https://github.com/actions/partner-runner-images) repo
